ui/frontend/reducers/code.ts

```diff
@@ -162,20 +166,141 @@ fn main() -> Result<(), Error> {
 }
 `;
 
+const SUDOKU: State = `use sunscreen::{
```

Comment:
[question]: Is there a way we can ensure this stays in sync with the checked in rust code running the example? It would be convenient to have one source of truth for the example (that can be built in CI); I'm not familiar enough with frontend to know if loading a github blob URL like https://github.com/Sunscreen-tech/Sunscreen/blob/main/examples/sudoku_zkp/src/main.rs would work.

Reply:
Huh, going through GH is an interesting idea. We'd just have to specify the version e.g. https://raw.githubusercontent.com/Sunscreen-tech/Sunscreen/v0.7.0/examples/pir/src/main.rs. I thought GH put raw content behind an expiring token, but maybe they stopped doing that.

It might require loading from the backend rather than frontend, but doesn't sound unreasonable.